### PR TITLE
highlight ExUnit assert/refute functions and macros

### PR DESCRIPTION
**Pull request by gasparch**
more syntax highlight for ExUnit tests

### CODE REVIEW

**Comment by kassio** [comment]
Can you add tests?

**Comment by gasparch** [comment]
tests added.

File: spec/syntax/exunit_spec.rb [hunk 1]
```diff
@@ -44,4 +44,126 @@
...
+  end
+
+  it 'refute_received ' do
+    expect(<<~EOF).to include_elixir_syntax('elixirExUnitAssert', 'refute_received')
+    test 'that stuff works' do
+      refute_received true
+    end
+    EOF
+  end
+
```
Comment (kassio) [inline spec/syntax/exunit_spec.rb]: ✂️ extra empty lines

File: spec/syntax/exunit_spec.rb [hunk 2]
```diff
@@ -44,4 +44,126 @@
...
+      end
+    end
+
+    test 'that stuff works' do
+      assert true
+    end
+    EOF
+  end
+
+  it 'assert ' do
```
Comment (kassio) [inline spec/syntax/exunit_spec.rb]: ✂️ extra empty space at the end of test descriptions.

**Comment by jbodah** [comment]
@gasparch I'd love to get this merged. Are you still working on this or would you like me to take over?

**Comment by gasparch** [comment]
@jbodah  thanks for prodding :) 

Did `commit --amend` to fix spacing issues.

**Pull request merged**